stl/inc/xstring

```diff
@@ -56,11 +56,15 @@ struct _Char_traits { // properties of a string or stream element
 
         return _Count;
     }
-

```

Comment:
This newline should be restored. We always separate functions with newlines, except in rare cases of very repetitive short functions.